README.md

```diff
@@ -197,14 +199,51 @@ class CustomPermission(BasePermission):
 
 Arguments:
 * `mutation`: mutation class
-* `info`: resolver info, whereas `info.context` is the [http request](https://docs.djangoproject.com/en/2.1/ref/request-response/#httprequest-objects) and user can be accessed through `info.context.user`
+* `info`: resolver info, whereas `info.context` is the [http request](https://docs.djangoproject.com/en/1.11/ref/request-response/#httprequest-objects) and user can be accessed through `info.context.user`
 * `instance`: instance being edited by specific mutation
 
 Save your permission module as `permissions.py` and inject it as Docker volume to path `/app/caluma/extensions/permissions.py`,
 see [docker-compose.yml](https://github.com/projectcaluma/caluma/blob/master/docker-compose.yml) for an example.
 
 Afterwards you can configure it in `PERMISSION_CLASSES` as `caluma.extensions.permissions.CustomPermission`.
 
+##### Validation classes
+
+Validation classes validate or amend input data of any mutation. It completes permission classes which allow execution of mutations based on updating object whereas validation classes
+process input data.
```

Comment:
Validation classes can validate or amend input data of any mutation. Each mutation is processed in two steps:

1. Permission classes check if a given mutation is allowed based on the object being mutated, and
2. Validation classes process (and potentially amend) input data of a given mutation